nvm.sh

```diff
@@ -1904,6 +1904,8 @@ nvm_install_source() {
   fi
 
   nvm_err "nvm: install ${VERSION} failed!"
+  # shellcheck disable=SC2031
```

Comment:
@ljharb I move that part out of the if, is that good for you? The function would be still the same.